### ISSUE TITLE
travis ci is broken with some cypress test

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#2500]
## Bug description & symptom

This tree cypress job failed on travis:
- workspace/content_dashboard_activity.js
- sidebar/presence_sidebar_dashboard.js
- app_html-document/operation_dashboard_dashboardbutton_create_html-document.js

Log visible for each failed:
~~~
1) operation :: workspace > create_new > html-document dashborad > button:

     TypeError: Cannot read property 'body' of null

      at getHostContenteditable (http://localhost:1337/__cypress/runner/cypress_runner.js:107034:29)

      at _getSelectionBoundsFromContentEditable (http://localhost:1337/__cypress/runner/cypress_runner.js:106952:29)

      at Module.getSelectionBounds (http://localhost:1337/__cypress/runner/cypress_runner.js:107274:14)

      at shouldUpdateValue (http://localhost:1337/__cypress/runner/cypress_runner.js:94473:81)

      at http://localhost:1337/__cypress/runner/cypress_runner.js:95196:14

      at Keyboard.performSimulatedDefault (http://localhost:1337/__cypress/runner/cypress_runner.js:95235:11)

      at Keyboard.simulatedKeydown (http://localhost:1337/__cypress/runner/cypress_runner.js:95137:25)

      at Keyboard.typeSimulatedKey (http://localhost:1337/__cypress/runner/cypress_runner.js:95160:12)

      at http://localhost:1337/__cypress/runner/cypress_runner.js:94895:17

      at tryCatcher (http://localhost:1337/__cypress/runner/cypress_runner.js:138967:23)

      at Function.Promise.attempt.Promise.try (http://localhost:1337/__cypress/runner/cypress_runner.js:136242:29)

      at http://localhost:1337/__cypress/runner/cypress_runner.js:94920:70

      at tryCatcher (http://localhost:1337/__cypress/runner/cypress_runner.js:138967:23)

      at Object.gotValue (http://localhost:1337/__cypress/runner/cypress_runner.js:138109:18)

      at Object.gotAccum (http://localhost:1337/__cypress/runner/cypress_runner.js:138098:25)

      at Object.tryCatcher (http://localhost:1337/__cypress/runner/cypress_runner.js:138967:23)
~~~
~~~
1) content :: home_page :

     CypressError: Timed out retrying: expected 'http://localhost:1337/ui' to include '/workspaces/1/dashboard'

      at Object.cypressErr (http://localhost:1337/__cypress/runner/cypress_runner.js:104940:11)

      at Object.throwErr (http://localhost:1337/__cypress/runner/cypress_runner.js:104895:18)

      at Object.throwErrByPath (http://localhost:1337/__cypress/runner/cypress_runner.js:104927:17)

      at retry (http://localhost:1337/__cypress/runner/cypress_runner.js:96263:16)

      at onFailFn (http://localhost:1337/__cypress/runner/cypress_runner.js:85359:16)

      at tryCatcher (http://localhost:1337/__cypress/runner/cypress_runner.js:138967:23)

      at Promise._settlePromiseFromHandler (http://localhost:1337/__cypress/runner/cypress_runner.js:136903:31)

      at Promise._settlePromise (http://localhost:1337/__cypress/runner/cypress_runner.js:136960:18)

      at Promise._settlePromise0 (http://localhost:1337/__cypress/runner/cypress_runner.js:137005:10)

      at Promise._settlePromises (http://localhost:1337/__cypress/runner/cypress_runner.js:137080:18)

      at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:1337/__cypress/runner/cypress_runner.js:133692:16)

      at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:1337/__cypress/runner/cypress_runner.js:133702:10)

      at Async.drainQueues (http://localhost:1337/__cypress/runner/cypress_runner.js:133576:14)
~~~
~~~
1) content :: workspace > dashbord dashboard__workspaceInfo > recentactivity:

     CypressError: Timed out retrying: Expected to find element: '.recentactivity .recentactivity__list .recentactivity__empty', but never found it.

      at Object.cypressErr (http://localhost:1337/__cypress/runner/cypress_runner.js:104940:11)

      at Object.throwErr (http://localhost:1337/__cypress/runner/cypress_runner.js:104895:18)

      at Object.throwErrByPath (http://localhost:1337/__cypress/runner/cypress_runner.js:104927:17)

      at retry (http://localhost:1337/__cypress/runner/cypress_runner.js:96263:16)

      at onFailFn (http://localhost:1337/__cypress/runner/cypress_runner.js:85359:16)

      at tryCatcher (http://localhost:1337/__cypress/runner/cypress_runner.js:138967:23)

      at Promise._settlePromiseFromHandler (http://localhost:1337/__cypress/runner/cypress_runner.js:136903:31)

      at Promise._settlePromise (http://localhost:1337/__cypress/runner/cypress_runner.js:136960:18)

      at Promise._settlePromise0 (http://localhost:1337/__cypress/runner/cypress_runner.js:137005:10)

      at Promise._settlePromises (http://localhost:1337/__cypress/runner/cypress_runner.js:137080:18)

      at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:1337/__cypress/runner/cypress_runner.js:133692:16)

      at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:1337/__cypress/runner/cypress_runner.js:133702:10)

      at Async.drainQueues (http://localhost:1337/__cypress/runner/cypress_runner.js:133576:14)
~~~
## Diagnostic

*Optional - explain what has been found and what is not yet clear. If unknown, write "unknown".*

## Expected

- [ ] cypress test working correctly

## Impacted components

*If unsure, write "unknown"*

**Comment by metzgegu** [comment]
Fixed tests in the new PR :

- [x] workspace/content_dashboard_activity.js 
- [ ] sidebar/presence_sidebar_dashboard.js **(Couldn't be reproduced)**
- [x] app_html-document/operation_dashboard_dashboardbutton_create_html-document.js

**Comment by PhilippeAccorsi** [comment]
Fixed in tracim 2.4.5
